src/sentry/relay/config/__init__.py

```diff
@@ -138,6 +138,7 @@ def get_project_config(project, full_config=True, project_keys=None):
 
     :return: a ProjectConfig object for the given project
     """
+    metrics.incr("sentry.relay.config.get_project_config")
```

Comment:
Do we care about absolute numbers here?
```suggestion
    metrics.incr("sentry.relay.config.get_project_config", sample_rate=1.0)
```